controllers/storageconsumer/storageconsumer_controller.go

```diff
@@ -164,6 +184,14 @@ func (r *StorageConsumerReconciler) reconcilePhases() (reconcile.Result, error)
 			return reconcile.Result{}, err
 		}
 
+		if err := r.reconcileCephClientCephFSProvisioner(); err != nil {
+			return reconcile.Result{}, err
+		}
+
+		if err := r.reconcileCephClientCephFSNode(); err != nil {
+			return reconcile.Result{}, err
+		}
+
 		if err := r.reconcileCephClientHealthChecker(); err != nil {
 			return reconcile.Result{}, err
 		}
```

Comment:
Can we rearrange the reconciles based on concerns,  RBD then CephFS (or the other way around)? 
One way will be:
```
r.reconcileCephClientRBDProvisioner
r.reconcileCephClientRBDNode
r.reconcileCephBlockPool
```

Then 
```
r.reconcileCephClientCephFSProvisioner
r.reconcileCephClientCephFSNode
r.reconcileCephFilesystemSubVolumeGroup
```

Then 
```
r.reconcileCephClientHealthChecker
```

This way we can guarantee that even if we have an error in CephFS we still have functioning Block storage and on top of that the code will be more ready for the next step where we will provide functionality based on request/caps